proj3_chess/player.py

```diff
@@ -0,0 +1,18 @@
+from typing import Callable
+
+
+class Player:
+    def __init__(self, direction: int, symbol: Callable[[str], str]) -> None:
```

Comment:
`direction` should be an Enum with `UP = 1` and `DOWN = -1`. Enforce the values you expect to see